### DISCUSSION TITLE
Continually encounter fatal error c1083 when attempting to install

### DISCUSSION THREAD

**Discussion by enicyy** [WyattBlue/auto-editor#586] [1. Help] [answered]
Hello,

Every time i try to install Auto-Editor using the command 'pip install auto-editor' I receive an error code and this line in my cmd

fatal error C1083: Cannot open include file: 'libavutil/mathematics.h': No such file or directory

Any help in resolving this would be greatly appreciated.

Thank you :-)

**Comment by WyattBlue** [comment, marked as answer]
This is a PyAV issue. Auto-Editor is pure python otherwise.

Easy option: Use the PyAV binary wheels if you're on a supported platform.
Hard option: Compile PyAV from source for whatever platform you're currently on.
